src/api/app/policies/workflow_run_policy.rb

```diff
@@ -1,9 +1,11 @@
 class WorkflowRunPolicy < ApplicationPolicy
-  class Scope
+  class Scope < Scope
     def initialize(user, scope, opts = {})
       @user = user
       @scope = scope
       @opts = opts
+
+      super(user, scope)
```

Comment:
I don't like this uncommented change to this policy inside a large commit. Please separate this commit and explain it in the commit message.

Reply:
@hennevogel I don't see a reason to touch the policy at all in this PR, I removed those changes.